ofono/bridge.py

```diff
@@ -0,0 +1,134 @@
+import logging
+import os
+from time import sleep
+
+import pydbus
+from dbus.mainloop.glib import DBusGMainLoop
+from gi.repository import GLib
+
+from helpers import Singleton, setup_logger
+
+DBusGMainLoop(set_as_default=True)  # has to be called first
+
+logger = setup_logger(__name__, 'debug')
+logging.basicConfig(level=logging.DEBUG)
+
+
+class OfonoBridge(object):
+    """
+    Generic util class to bridge between ZPUI and ofono backend through D-Bus
+    """
+
+    def __init__(self):
+        super(OfonoBridge, self).__init__()
+
+        bus = pydbus.SystemBus()
+        manager = bus.get('org.ofono', '/')
+        modem_path = manager.GetModems()[0][0]
+
+        if modem_path != '/sim900_0':
+            raise ValueError("Default modem should be '/sim900_0', was '{}'".format(modem_path))
+
+        # self.start()  #todo: check if it's the suitable place to do it
```

Comment:
I don't think so - it makes sense not to start() the loop right when the OfonoBridge is initialized.

Reply:
As in, explicit is better than implicit =)